explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -570,12 +569,24 @@ object ObsTabTiles:
             val tiles =
               alltiles.filterNot(t => removedIds.contains(t.id))
 
-            TileController(
-              props.vault.userId,
-              props.resize.width.getOrElse(0),
-              defaultLayout,
-              layout,
-              tiles,
-              section,
-              props.backButton.some
+            React.Fragment(
+              TileController(
+                props.vault.userId,
+                props.resize.width.getOrElse(0),
+                defaultLayout,
+                layout,
+                tiles,
+                section,
+                props.backButton.some
+              ),
+              TileController(
```

Comment:
So you can't drag it to the other controller right?

Reply:
That is correct.